stories/3-sorting-bug.md

```diff
@@ -8,4 +8,4 @@
 ## Feature Requirements
 
 1. When displaying the list of members, they should be sorted alphabetically ascending by last name then first name. (eg "Victor Melo" then "Abigail Miller" then "Brandon Miller")
```

Comment:
```suggestion
1. When displaying the list of members, they should be sorted alphabetically A-Z by last name then first name. (eg "Victor Melo" then "Abigail Miller" then "Brandon Miller")
```
Might be more clear?

Reply:
I'd add `Victor sorts first because Melo sorts before Miller, and then Abigail sorts before Brandon, but both Abigail and Brandon sort after Melo due to their last name Milller`

Reply:
I just felt like it a bit more context would be good